src/components/web/home/home-screen-web.scss

```diff
@@ -49,34 +50,37 @@
     }
   }
 
-  .play {
+  .button {
     width: 300px;
     height: 60px;
     margin-top: 5px;
     font-size: 2rem;
+    background-color: white;
 
     transition: all 0.1s linear;
     animation: slideInUp 0.4s cubic-bezier(0.175, 0.885, 0.32, 1.05) 0.8s;
     animation-fill-mode: backwards;
 
-    &:hover {
-      margin-top: 0px;
-      width: 310px;
-      height: 65px;
-      font-size: 2.1rem;
-      cursor: pointer;
+    &.loading {
```

Comment:
by doing this outside the button.scss, it means you'll have to do it again for every button that loads. What to do is:

- in Button.tsx, apply a className 'loading' if you're loading (you're kind of doing this, but shouldn't prevent their custom classname from being added, so make it a separate line)
- then in button.scss, you can say the above; if loading, don't do hover styles